packages/e2e-tests/src/tests/SessionSigs.spec.ts

```diff
@@ -0,0 +1,825 @@
+      await executeJsSigningParallel(devEnv, getLitActionSessionSigs);
+    });
+
+    it('LitAction IPFS Session', async () => {
+      await executeJsSigningParallel(
+        devEnv,
+        getLitActionSessionSigsUsingIpfsId
+      );
+    });
+
+    it('EOA Wallet', async () => {
+      await executeJsSigningParallel(devEnv, getEoaSessionSigs);
+    });
+
+    it('PKP Session', async () => {
+      await executeJsSigningParallel(devEnv, getPkpSessionSigs);
+    });
+  });
+
+  describe('Broadcast And Collect', () => {
+    it('LitAction Session', async () => {
+      await broadcastAndCollect(devEnv, getLitActionSessionSigs);
+    });
+
+    it('LitAction IPFS Session', async () => {
+      await broadcastAndCollect(devEnv, getLitActionSessionSigsUsingIpfsId);
+    });
+
+    it('EOA Wallet', async () => {
+      await broadcastAndCollect(devEnv, getEoaSessionSigs);
+    });
+
+    it('PKP Session', async () => {
+      await broadcastAndCollect(devEnv, getPkpSessionSigs);
+    });
+  });
+
+  describe('Decrypt And Combine', () => {
+    it('LitAction Session', async () => {
+      await decryptAndCombine(devEnv, getLitActionSessionSigs);
+    });
+
+    it('LitAction IPFS Session', async () => {
+      await decryptAndCombine(devEnv, getLitActionSessionSigsUsingIpfsId);
+    });
+
+    it('EOA Wallet', async () => {
+      await decryptAndCombine(devEnv, getEoaSessionSigs);
+    });
+
+    it('PKP Session', async () => {
+      await decryptAndCombine(devEnv, getPkpSessionSigs);
+    });
+  });
+
+  describe('Sign And Combine ECDSA', () => {
+    it('LitAction Session', async () => {
+      await signAndCombine(devEnv, getLitActionSessionSigs);
+    });
+
+    it('LitAction IPFS Session', async () => {
+      await signAndCombine(devEnv, getLitActionSessionSigsUsingIpfsId);
+    });
+
+    it('EOA Wallet', async () => {
+      await signAndCombine(devEnv, getEoaSessionSigs);
+    });
+
+    it('PKP Session', async () => {
+      await signAndCombine(devEnv, getPkpSessionSigs);
+    });
+  });
+
+  it('Invalid lit action Custom Auth SessionSigs', async () => {
+    devEnv.setUnavailable(LIT_TESTNET.MANZANO);
+
+    const alice = await devEnv.createRandomPerson();
+
+    try {
+      await getInvalidLitActionSessionSigs(devEnv, alice);
+    } catch (e: any) {
+      console.log('❌ This error is expected', e);
+      if (
+        e.message ===
+        'There was an error getting the signing shares from the nodes'
+      ) {
+        console.log(
+          '✅ testUseInvalidLitActionCodeToGenerateSessionSigs passed'
+        );
+      } else {
+        throw e;
+      }
+    } finally {
+      devEnv.releasePrivateKeyFromUser(alice);
+    }
+  });
+
+  it('Invalid Lit Action Custom Auth IPFS SessionSigs', async () => {
+    const alice = await devEnv.createRandomPerson();
+
+    try {
+      await getInvalidLitActionIpfsSessionSigs(devEnv, alice);
+    } catch (e: any) {
+      console.log('❌ THIS IS EXPECTED: ', e);
```

Comment:
As above, We can use `expect().toThrow(...)` here, which will let us :hocho: the try/catch entirely and get rid of the `console.logs`. The log on L242 is particularly confusing because we don't yet _know_ that it's actually expected, and if it _was_ expected we see a big red X -- ideally all of this would be condensed into a single `.toThrow()` check